collectors/meta_template/schema.json

```diff
@@ -0,0 +1,393 @@
+                    "description": "Title for the prerequisites sub-section"
+                  },
+                  "description": {
+                    "type": "string",
+                    "description": "Description for the prerequisites sub-section, you can use lists or any other markdown-like syntax"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "configuration": {
+          "type": "object",
+          "properties": {
+            "file": {
+              "type": "object",
+              "description": "Specify in a text key which file the user has to edit.",
+              "properties": {
+                "description": {
+                  "type": "string",
+                  "description": "Example: go.d/apache.conf"
+                }
+              }
+            },
+            "options": {
+              "type": "object",
+              "properties": {
+                "description": {
+                  "type": "string",
+                  "description": "If there is anything specific about this section that needs to be noted mention it here."
+                },
+                "folding": {
+                  "type": "object",
+                  "description": "Whether or not to collapse the options table. If set to true a title must be also provided",
+                  "properties": {
+                    "title": {
+                      "type": "string",
+                      "description": "Title for the collapsed section.\n Example: Config options"
+                    },
+                    "enabled": {
+                      "type": "boolean",
+                      "description": "boolean value that controls whether to collapse section or not."
+                    }
+                  }
+                },
+                "list": {
+                  "type": "array",
+                  "description": "Each of the list's items represents a row in the available config options table",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "name": {
+                        "type": "string",
+                        "description": "Option name"
+                      },
+                      "description": {
+                        "type": "string",
+                        "description": "Option description"
+                      },
+                      "default_value": {
+                        "type": [
+                          "string",
+                          "number"
+                        ],
+                        "description": "Default value of the option, leave empty if none"
+                      },
+                      "required": {
+                        "type": "boolean",
+                        "description": "A boolean value indicating if this options is required or not. Use true / false"
+                      }
```

Comment:
Option "details" is missing see [an example](https://github.com/netdata/go.d.plugin/blob/b2b7637c59ad51f98eaec93285890fb6d7b76900/modules/openvpn/metadata.yaml#L49-L62)

Reply:
Thanks, didn't have this field in my template, updated the schema.